README.md

```diff
@@ -8,6 +8,11 @@ This is a vim/gvim extension for coala.
 Simply save your file after loading it and you'll have coala running
 automatically and giving you the output in a popup.
 
+State
+=====
+
+This plugin is rather young. It is usable but beware of bugs! And file them.
```

Comment:
I've seen ``please`` being used quite often in this situation. imo you should use : `` And please file them!``